### DISCUSSION TITLE
file_dialog_save with extension filter

### DISCUSSION THREAD

**Discussion by xarthurx** [libigl/libigl#1794] [Q&A] [answered]
when using `file_dialog_save()`  to save files, is it possible to add extension filter so that the file is saved in certain format by default?

![image](https://user-images.githubusercontent.com/1921878/116114944-259b6180-a6ba-11eb-8491-068a7f975c46.png)

**Comment by jdumas** [comment, marked as answer]
This is currently not supported, but I invite you to try out more complete third-party libraries that offer those functionalities:
- https://sourceforge.net/projects/tinyfiledialogs/
- https://github.com/btzy/nativefiledialog-extended

**Reply by xarthurx** [reply]
Thank you for the always-very-responsive reply!
